.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "deps/lua-memory"]
+	path = deps/lua-memory
+	url = https://github.com/MatheusNtg/lua-memory
```

Comment:
it should refers to upstream, right? or do you have modifications in your fork?

Reply:
I have modifications on my fork.

Reply:
so, you need to make a PR as well.. what's the upstream repo? I think we should fork it on luainkernel and make a PR for it..

Reply:
The upstream repo can be found [here](https://github.com/luainkernel/lua-memory). I made a PR to lua-memory repo now